subsys/bluetooth/audio/pacs.c

```diff
@@ -256,6 +258,20 @@ static int set_available_contexts(uint16_t contexts, uint16_t *available,
 	return 0;
 }
 
+static int set_supported_contexts(uint16_t contexts, uint16_t *supported)
+{
+	int err;
+
+	*supported = contexts;
```

Comment:
In case that `*supported == contexts` we should not send a notification

Reply:
Done